tests/test_configuration.py

```diff
@@ -83,16 +83,6 @@ def test_configuration(self):
         # let's also add a mannings to the domain (constant for this example)
         mesh.mannings_n_at_sea_floor = numpy.full(mesh.values.shape, 0.025)
 
-        # init tidal forcing and setup requests
-        tidal_forcing = Tides()
-        tidal_forcing.use_constituent('M2')
-        tidal_forcing.use_constituent('N2')
-        tidal_forcing.use_constituent('S2')
-        tidal_forcing.use_constituent('K1')
-        tidal_forcing.use_constituent('O1')
-
-        mesh.add_forcing(tidal_forcing)
-
```

Comment:
I removed tidal forcing for now so it doesn't download TPXO, but in the future we could use another call of https://github.com/actions/cache just like we currently do for dependencies